src/screens/DevicesScreen.js

```diff
@@ -23,6 +24,15 @@ type InjectedProps = {|
 @errorBoundary(<ErrorScreen showBackButton />)
 @observer
 class DevicesScreen extends InjectedComponent<InjectedProps> {
```

Comment:
DeviceDetailsScreen is always on stack after DevicesScreen, so making this only in one place should be enough